src/homework2/Arrays/Task11.java

```diff
@@ -0,0 +1,63 @@
+package homework2.Arrays;
+
+import java.util.Arrays;
+
+public class Task11 {
+    public static void main(String[] args) {
+        // ---------------------------------------------------------
+        // task - 11: We have two one-dimensional arrays with natural numbers. Create a program that compares all numbers
+        //    with the same indexes from both arrays and writes in a third array the smaller of the two numbers. At the end print the console the following:
+        //    -	the content of all three arrays
+        //    -	the sum of the three arrays
+        //    -	the product of the elements of the last array
+        // ---------------------------------------------------------
+
+        int[] firstArr = { 18, 19, 32, 1, 3, 4, 5, 6, 7, 8 };
+        int[] secondArr = { 1, 2, 3, 4, 5, 16, 17, 18, 27, 11 };
+
+        if (firstArr.length != secondArr.length) {
+            System.out.println("Arrays cannot be compared.");
+        } else {
+            int[] minValueArr = new int[firstArr.length];
+
+            for (int i = 0; i < minValueArr.length; i++) {
+                if (firstArr[i] > secondArr[i]) {
```

Comment:
for (int index = 0; index < array3.length; index++) {
            array3[index] = Math.min(array1[index], array2[index]);
        }